core/tests/protractor_desktop/wipeout.js

```diff
@@ -0,0 +1,110 @@
+    await deleteAccountPage.requestAccountDeletion();
+    expect(await browser.getCurrentUrl()).toEqual(
+      'http://localhost:9001/pending-account-deletion');
+  });
+
+  it('should delete private exploration', async function() {
+    await users.createUser('voiceArtist@oppia.com', 'voiceArtist');
+    await users.createAndLoginUser('user2@delete.com', 'userToDelete2');
+    await workflow.createExploration();
+    var explorationId = await general.getExplorationIdFromEditor();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('voice artists');
+    await workflow.addExplorationVoiceArtist('voiceArtist');
+    await deleteAccountPage.get();
+    await deleteAccountPage.requestAccountDeletion();
+    expect(await browser.getCurrentUrl()).toEqual(
+      'http://localhost:9001/pending-account-deletion');
+    await users.logout();
+    await users.login('voiceArtist@oppia.com');
+    await general.openEditor(explorationId);
+    await general.expect404Error();
+  });
+
+  it('should set published exploration as community owned', async function() {
+    await users.createUser('user@check.com', 'userForChecking');
+    await users.createAndLoginUser('user3@delete.com', 'userToDelete3');
+    await workflow.createAndPublishExploration(
+      EXPLORATION_TITLE,
+      EXPLORATION_CATEGORY,
+      EXPLORATION_OBJECTIVE
+    );
+    var explorationId = await general.getExplorationIdFromEditor();
+    await deleteAccountPage.get();
+    await deleteAccountPage.requestAccountDeletion();
+    expect(await browser.getCurrentUrl()).toEqual(
+      'http://localhost:9001/pending-account-deletion');
+    await users.logout();
+    await users.login('user@check.com');
+    await general.openEditor(explorationId);
+    await workflow.isExplorationCommunityOwned();
+  });
+
+  it('should keep published exploration with other owner', async function() {
+    await users.createUser('secondOwner@check.com', 'secondOwner');
+    await users.createAndLoginUser('user4@delete.com', 'userToDelete4');
+    await workflow.createExploration();
+    var explorationId = await general.getExplorationIdFromEditor();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('second owner');
+    await workflow.addExplorationManager('secondOwner');
+    await deleteAccountPage.get();
+    await deleteAccountPage.requestAccountDeletion();
+    expect(await browser.getCurrentUrl()).toEqual(
+      'http://localhost:9001/pending-account-deletion');
+    await users.logout();
+    await users.login('secondOwner@check.com');
+    await general.openEditor(explorationId);
+    await explorationEditorPage.navigateToSettingsTab();
+    expect(await workflow.getExplorationManagers()).toEqual(['secondOwner']);
+  });
+
+  afterEach(async function() {
+    await general.checkForConsoleErrors(
+      ['Failed to load resource: the server responded with a status of 404']);
```

Comment:
This way we won't be sure which `it` block raised this issue! maybe have an expectedConsoleError variable and clear it in beforeEach and populate it in the specific it block, wdyt?

Reply:
Done.